molr-mole-core/src/main/java/io/molr/mole/core/runnable/lang/OngoingExecutionStrategyConfiguration.java

```diff
@@ -0,0 +1,53 @@
+package io.molr.mole.core.runnable.lang;
+
+import java.util.Set;
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableSet;
+
+import io.molr.commons.domain.ExecutionStrategy;
+
+public class OngoingExecutionStrategyConfiguration {
+
+	
+	private final ExecutionStrategyConfiguration.Builder builder;
+	
+	OngoingExecutionStrategyConfiguration(ExecutionStrategyConfiguration.Builder builder) {
+		requireNonNull(builder);
+		this.builder = builder;
+	}
+	
+	/**
+	 * Sets the default {@link ExecutionStrategy} for this mission. If none default {@link ExecutionStrategy} is specified molr will either use the global default
+	 * or the one and only allowed strategy if and only if exactly one strategy has been specified by {@link #allowed}. If multiple allowed strategies have been specified
+	 * the default strategy must be a member of the allowed strategies.
+	 *
+	 * @param executionStrategy
+	 * @return
```

Comment:
I think this empty parameter and  @return statments might be a problem for the travis build (and in general should be filled ;-)